src/modules/commander/accelerometer_calibration.cpp

```diff
@@ -175,7 +175,7 @@ typedef struct  {
 
 int do_accel_calibration(orb_advert_t *mavlink_log_pub)
 {
-#ifndef __PX4_QURT
+#if !defined(__PX4_QURT) && !defined(__PX4_POSIX_EAGLE)
```

Comment:
I didn't think it would work on the DSP, since it seemed to require access to an IP stack.  I was also thinking that the console was maintained in this module, which is something that the DSP doesn't support.  Is this an incorrect assumption on my part?